pkg/kapis/devops/v1alpha4/register.go

```diff
@@ -0,0 +1,38 @@
+package v1alpha4
+
+import (
+	"context"
+	"github.com/emicklei/go-restful"
+	"kubesphere.io/devops/pkg/api"
+	"kubesphere.io/devops/pkg/api/devops/v1alpha4"
+	"kubesphere.io/devops/pkg/apiserver/runtime"
+	"kubesphere.io/devops/pkg/client/k8s"
+	"net/http"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type Option struct {
+	Container *restful.Container
+	K8sClient k8s.Client
+	Client    client.Client
+}
+
+func AddToContainer(o Option) error {
+	// TODO Complete definitions of route
+	//ws := runtime.NewWebService(v1alpha4.GroupVersion)
+	//ws.Route(ws.GET("/devops/{devops}/credentials"))
+	ws := runtime.NewWebServiceWithoutGroup(v1alpha4.GroupVersion)
+	// TODO Remove this test API
```

Comment:
Are you sure this PR is ready?

Reply:
hi @LinuxSuRen , this PR has ready for review. Thank you for your patience.